### DISCUSSION TITLE
How to build hasura graphql query with join for condition?

### DISCUSSION THREAD

**Discussion by V-ampire** [hasura/graphql-engine#9771] [Question]
I have 2 tables with - users and users_ban, so I want to make graphql using hasura server query which fetch users and join boolean field either user in ban list, like in sql:

```
select 
id, 
email,
case
  when users_ban.id is null then false
  else true
end is_banned
from users
left join users_ban
on users.id = users_ban.user_id;
```
How can I do it via graphql and hasura server?

**Comment by SamirTalwar** [comment]
@V-ampire, have you tried [computed fields](https://hasura.io/docs/latest/schema/postgres/computed-fields/)?

**Reply by V-ampire** [reply]
Nope, I will look into it, but I'd like to find a way to avoid modify backed and fecth data using query only. Is this possible?

**Reply by SamirTalwar** [reply]
I suggest using [native queries](https://hasura.io/docs/latest/schema/postgres/logical-models/native-queries/) then.

**Reply by V-ampire** [reply]
I see, thanks, I thought it's possible to use only graphql queries
